crawl-ref/source/makeitem.cc

```diff
@@ -1612,24 +1612,22 @@ static bool _try_make_jewellery_unrandart(item_def& item, int force_type,
 /**
  * Generate a random 'bad' plus for a ring type that cares about plusses.
  *
- * @return a bad 'plus', between -2 and -6 (inclusive).
+ * @return a bad 'plus', between -3 and -4 (inclusive).
```

Comment:
Just make bad rings always `-4` and delete this whole function.

Reply:
Currently, 50% of rings are -2, 33% of rings are -3, and 17% of rings are -4 or more.  Because t average cursed ring is -2.83, i'm inclined to pick -3 if i'm going to pick a static number.  But maybe you're suggesting bumping it up to -4 because cursed items aren't meaningfully punishing at present?

Reply:
Cursed items are pretty vestigial in crawl at this point, but yes, as long as they still exist they should be more impactful.